code/infrastructure-api/src/stacks/jupyterStack.js

```diff
@@ -6,17 +6,18 @@ import serviceApi from '../kubernetes/serviceApi';
 import proxyRouteApi from '../kong/proxyRouteApi';
 import { createDeployment, createService, createProxyRoute } from './stackBuilders';
 
-function createJupyterNotebook(datalabInfo, name, type) {
+function createJupyterNotebook(params) {
+  const { datalabInfo, name, type } = params;
```

Comment:
The create and delete functions are a little inconsistent with the use of destructuring.